Calculator/Calculator/Model/Enumeration/ExpressionParser.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ExpressionParser.swift
+//  Calculator
+//
+//  Created by jiye Yi on 2023/01/27.
+//
+
+enum ExpressionParser {
+    static func parse(from input: String) -> Formula {
+        let components = componentsByOperator(from: input)
+        
+        let operand: [Double] = components.compactMap{ Double($0) }
+        let operators: [Operator] = components.filter{ $0.count == 1 }
+            .compactMap{ Operator(rawValue: Character($0)) }
```

Comment:
중괄호 전에도 띄어쓰기 넣어주세요~!
```suggestion
        let operand: [Double] = components.compactMap { Double($0) }
        let operators: [Operator] = components.filter { $0.count == 1 }
            .compactMap { Operator(rawValue: Character($0)) }
```